modular/gater/auth_handler.go

```diff
@@ -26,6 +26,17 @@ const (
 	ExpectedEddsaPubKeyLength int    = 64
 )
 
+type RequestNonceResp struct {
+	CurrentNonce     int32  `xml:"currentNonce"`
```

Comment:
Should XML variable names start with an uppercase letter?
https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html
<img width="758" alt="image" src="https://github.com/bnb-chain/greenfield-storage-provider/assets/67917837/62ab9480-3074-4e3b-8151-07e18e35b44c">

![Uploading image.png…]()